index.html

```diff
@@ -494,6 +494,9 @@ <h4 style="margin-top: 15px;">Follow Us</h4>
                     <li><a href="#" class="social"><i class="fa-brands fa-facebook-f"></i> Facebook</a></li>
                     <li><a href="#" class="social"><i class="fa-brands fa-instagram"></i> Instagram</a></li>
                     <li><a href="#" class="social"><i class="fa-brands fa-linkedin-in"></i> Linkedin</a></li>
+                    <li><a href="#" class="social"><i class="fab fa-youtube"></i> Youtube</a></li>
+                    <li><a href="#" class="social"><i class=" fab fa-twitter"></i> Twitter</a></li>
```

Comment:
The Twitter icon has been updated. Please replace the existing icon with the latest version to ensure consistency.